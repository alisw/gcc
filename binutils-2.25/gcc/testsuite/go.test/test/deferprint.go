../../../../../gcc/testsuite/go.test/test/deferprint.go