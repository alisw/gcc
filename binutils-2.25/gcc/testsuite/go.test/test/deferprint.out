../../../../../gcc/testsuite/go.test/test/deferprint.out